.github/workflows/gh-pages.yml

```diff
@@ -41,6 +41,8 @@ jobs:
           cd viz
           pnpm install
       - name: Build
+        env:
+          VITE_RUNPOD_API_KEY: ${{ secrets.VITE_RUNPOD_API_KEY }}
```

Comment:
TODO: Add this as a secret in github actions

Reply:
Added!